docs/hw/display/nv3/pcrtc.rst

```diff
@@ -43,3 +44,74 @@ Vertical/horizontal blanking signals
 ====================================
 
 .. todo:: write me
+
+RMA access port
+===============
+
+.. reg:: 32 nv3-io-rma-access RMA access port
+
+This is a 32-bit port that writes into the RMA address specified by CRTC register 0x38.
+
+CRTC registers
+==============
+
+.. reg:: 8 nv3-io-crtc-index CRTC index
+
+.. reg:: 8 nv3-io-crtc-data CRTC data
+
+.. space:: 8 nv3-crtc-ext-regs 0x100 Extended VGA registers
+
+   0x19 REPAINT_0 nv3-crtc-ext-rpc0
+   0x1a REPAINT_1 nv3-crtc-ext-rpc1
+   0x1d WRITE_BANK nv3-crtc-ext-write-bank
+   0x1e READ_BANK nv3-crtc-ext-read-bank
+   0x25 EXTENDED_VERT nv3-crtc-ext-extvert
+   0x28 PIXEL_FMT nv3-crtc-ext-pixel-fmt
+   0x2d EXTENDED_HORZ nv3-crtc-ext-exthorz
+   0x38 RMA_MODE nv3-crtc-ext-rmamode
+   0x3e I2C_READ nv3-crtc-ext-i2c-read
+   0x3f I2C_WRITE nv3-crtc-ext-i2c-write
+
+.. reg:: 8 nv3-crtc-ext-rpc0 Extended Start Address and Row Offset
+
+Bits 4-0 are Start Address bits 16-20. Bits 7-5 are Row Offset bits 3-5.
```

Comment:
This should be indented (we use 3 spaces in envytools), so that it becomes part of the .. reg directive